rpa-manager/modules/ROOT/pages/processautomation-prepare.adoc

```diff
@@ -286,11 +293,21 @@ Check the usage of a credential before changing or deleting it.
 
 ==== Create Credentials
 
-Create credentials with which a bot can login to an external application during process runs:
+Create credentials with which a bot can login to an external application during process runs.
+
+Use descriptive names for credentials and only use them in an appropriate context. If, for example, the employee `Nina` has an account for two applications, `Letters` and `Numbers`, create either two credentials (`nina_account_letters` and `nina_account_numbers`) or a single one that includes both applications in its name (`nina_accounts_letters_numbers`).
+
+To create a credential:
 
 . Open the *Credential Pool* view of the *Process Automation* module.
 . Click *Create*.
 . [[form-create-credential]] Complete the form *Create a New Credential*:
+* [[form-create-credential-useraffiliation]] *User affiliation*:
++
+Click to choose users that may view, edit, and use the credential. Add yourself to be able to view, edit, and use the credential, too.
+* *User group affiliation*:
++
+Click to choose user groups whose users may view, edit, and use the credential.
```

Comment:
```suggestion
Click to choose user groups whose users can view, edit, and use the credential.
```

Reply:
I left 'may' because it's about permissions.